pkg/runner/run_context.go

```diff
@@ -87,6 +88,18 @@ func (rc *RunContext) jobContainerName() string {
 	return createContainerName("act", rc.String())
 }
 
+// networkName return the name of the network which will be created by `act` automatically for job,
+// only create network if using a service container
+func (rc *RunContext) networkName() (string, bool) {
```

Comment:
What should this function return for the service container network if `--network` is specified? Should the service containers also use the user-supplied network?